blog/2023-06-28-pulsar-summit-north-america-2023-cfp.md

```diff
@@ -0,0 +1,56 @@
+
+Since 2020, seven global Pulsar Summit Events have featured 170+ interactive sessions by tech leads, open-source developers, software engineers, and software architects from Google, AWS, Splunk, Tencent, Verizon Media, Iterable, Yahoo, Nutanix, BIGO, TIBCO, OVHcloud, Clever Cloud, and more. The conferences have garnered 2,200 attendees representing 700 companies, including individuals from leading organizations such as Google, Microsoft, AMEX, Salesforce, TikTok, Alibaba, Tencent, Disney, and Paypal.
+
+## Speak at Pulsar Summit North America 2023!
+
+[Submit your session abstract](https://sessionize.com/pulsar-summit-north-america-2023)
+
+Share your Pulsar story and speak at the summit! Pulsar Summit offers a unique opportunity to connect with your peers and raise your profile in the rapidly growing Apache Pulsar community. 
+
+Our theme this year is "Why Pulsar?" and we are looking for stories that are innovative, informative, or thought-provoking. Join us to speak at the summit! You will be on stage with all the top Pulsar thought-leaders. It is a great way to participate and raise your profile in the rapidly growing Apache Pulsar community.
+
+#### As a speaker, you will receive:
+
+* Free conference pass.
+* Your headshot, bio and session featured on the Pulsar Summit website.
+* Your session will be promoted on YouTube, Twitter and LinkedIn.
+* The opportunity to share your knowledge and engage with the vibrant Pulsar community!
+
+**To speak at the summit, please submit an abstract about your presentation.** Remember to keep your proposal short, relevant, and engaging. All levels of talks (beginner, intermediate, and advanced) are welcome. We invite you to submit your talk proposals and share your unique experience in one of the following themes:
+
+* **“Why Pulsar?”** - Your journey is inspiring, and we want to hear it! Share your stories of why you chose Pulsar, the challenges you overcame, and the solutions you implemented. Let others learn from your experiences and gain valuable insights from your use cases.
+* **“Learning Pulsar”** - Are you an educator at heart? Submit an entry-level talk introducing Pulsar, its remarkable features, and best practices. Help beginners navigate their Pulsar journey and inspire them with your expert knowledge.
+* **“Deep Dive”** - For all the experts out there, we invite you to share deep technical knowledge about Pulsar. Whether it's about the inner workings, optimization techniques, or advanced features, we'd love to hear it all.
+* **“Around Pulsar”** - Have you built exceptional tools to work with Pulsar? Or have you integrated Pulsar with other technologies to create a superior solution? We'd love to hear about your innovations.
+
+## Important Dates:
+
+* CFP opens: Tuesday, June 21st, 2023
+* CFP closes: Friday, July 7th, 2023
+* Speaker notifications sent: Friday, July 28th, 2023
+* Schedule announcement: August 4th, 2023
```

Comment:
Since you added the specific day for others, perhaps you can add it here for "Schedule announcement" for consistency.

`August 4th, 2023` -> `Friday, August 4th, 2023`

Reply:
```suggestion
* Schedule announcement: Friday, August 4th, 2023
```